lib/as-text-plugin.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { promises as fs } from 'fs';
+
+const prefix = 'as-text:';
```

Comment:
super super nitpicky, but FWIW I believe most of the other bundlers/etc use simple `text:` for this:
https://docs.google.com/document/d/1BObemtyKibEYKZPuu5YqHlam6gRSA42oqPMnuSGHcTs/edit#heading=h.da1uub3uzevq

Reply:
Makes sense. I'll switch it over.